From 99302dc7640ae4840c1ec64a4ac7df703c4d4b11 Mon Sep 17 00:00:00 2001
From: Olivier Fourdan <ofourdan@redhat.com>
Date: Tue, 23 Jul 2024 17:11:55 +0200
Subject: [PATCH 001/105] build: Drop libxcvt requirement from
 SDK_REQUIRED_MODULES

The SDK doed not need libxcvt, only Xorg and Xwayland do.

Closes: https://gitlab.freedesktop.org/xorg/xserver/-/issues/1721
Fixes: a4ab57cb7 - build: Add dependency on libxcvt
Signed-off-by: Olivier Fourdan <ofourdan@redhat.com>
Part-of: <https://gitlab.freedesktop.org/xorg/xserver/-/merge_requests/1618>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index f8145e4a9..7d442c214 100644
--- a/configure.ac
+++ b/configure.ac
@@ -763,7 +763,7 @@ PKG_CHECK_MODULES(PIXMAN, $LIBPIXMAN)
 REQUIRED_LIBS="$REQUIRED_LIBS $LIBPIXMAN $LIBXFONT xau"
 
 dnl Core modules for most extensions, et al.
-SDK_REQUIRED_MODULES="$XPROTO $RANDRPROTO $RENDERPROTO $XEXTPROTO $INPUTPROTO $KBPROTO $FONTSPROTO $LIBPIXMAN $LIBXCVT"
+SDK_REQUIRED_MODULES="$XPROTO $RANDRPROTO $RENDERPROTO $XEXTPROTO $INPUTPROTO $KBPROTO $FONTSPROTO $LIBPIXMAN"
 # Make SDK_REQUIRED_MODULES available for inclusion in xorg-server.pc
 AC_SUBST(SDK_REQUIRED_MODULES)
 
-- 
2.45.2

